packages/moonstone/Dropdown/DropdownList.js

```diff
@@ -145,26 +149,20 @@ const DropdownListSpotlightDecorator = hoc((config, Wrapped) => {
 			this.scrollIntoView();
 		}
 
-		getNodeToFocus = (selected) => {
-			const scrollIndex = selected > scrollOffset ? selected - scrollOffset : 0;
-
-			return {
-				nodeToFocus: this.node.querySelector(`[data-index='${selected}']`),
-				nodeToScroll: this.node.querySelector(`[data-index='${scrollIndex}']`)
-			};
-		}
-
 		getScrollTo = (scrollTo) => {
 			this.scrollTo = scrollTo;
 		}
 
 		scrollIntoView = () => {
+			const {selected} = this.props;
+
 			if (!this.isScrolledIntoView) {
 				if (isSelectedValid(this.props)) {
-					const {nodeToFocus, nodeToScroll} = this.getNodeToFocus(this.props.selected);
+					const scrollIndex = selected > scrollOffset ? selected - scrollOffset : 0;
+					const selectedNode = this.node.querySelector(`[data-index='${selected}']`);
```

Comment:
Is this able to find one that's not currently in the DOM?  Should we make this a callback on the scroll, maybe?

Reply:
good question. I'll give it a try with a bigger data set and see.

Reply:
Fixed up (I hope!)

Reply:
Fixed up (I hope!)